\documentclass[11pt]{article}
\usepackage{scrtime} % for \thistime (this package MUST be listed first!)
\usepackage{graphicx}
\usepackage{float}
\usepackage[margin=0.75in]{geometry}
\usepackage{fancyhdr}
\usepackage{caption}
\usepackage[super]{natbib}
\usepackage[hidelinks]{hyperref} %hidelinks will remove ugly coloured links in the text
%\usepackage{underscore}
\usepackage{pdfpages}
\usepackage{xcolor,colortbl}%for changing cell colour
\usepackage{multirow}
\usepackage[normalem]{ulem}
\useunder{\uline}{\ul}{}
\usepackage{xspace}
\usepackage{longtable}
\usepackage{booktabs}
\usepackage{capt-of}
\pagestyle{fancy}
\setlength{\headheight}{15.2pt}
\setlength{\headsep}{13 pt}
\setlength{\parindent}{28 pt}
\setlength{\parskip}{12 pt}
\pagestyle{fancyplain}
\usepackage[T1]{fontenc}
\usepackage{tikz-cd}
\usepackage{tikz}
\usetikzlibrary{decorations.markings}
\usetikzlibrary{calc, arrows}
\usepackage{color,amsmath,amssymb,amsthm,mathrsfs,amsfonts,dsfont}
\lhead{Spatial Patterns of Substitutions in Bacterial Genomes}
%\rhead{Daniella Lato \thepage\ }
\rhead{Daniella Lato}
\title{\sc Something COG}
\author{\sc Daniella Lato \\ \sc Paper Draft}
%\date{\sc Committee Members: \\ \sc Dr. Brian Golding \\ \sc Dr. Marie Elliot \\ Transfer Exam Chair: \\ Dr. Ben Evans}
\renewcommand\headrulewidth{0.5mm}
\newcommand{\s}{\textit{Sinorhizobium}\xspace}
\newcommand{\sm}{\textit{Sinorhizobium meliloti}\xspace}
\newcommand{\smel}{\textit{S.\,meliloti}\xspace}
\newcommand{\smed}{\textit{S.\,medicae}\xspace}
\newcommand{\sfred}{\textit{S.\,fredii}\xspace}
\newcommand{\ssah}{\textit{S.\,saheli}\xspace}
\newcommand{\ster}{\textit{S.\,terangae}\xspace}
\newcommand{\agrot}{\textit{Agrobacterium tumefaciens}\xspace}
\newcommand{\agro}{\textit{A.\,tumefaciens}\xspace}
\newcommand{\bur}{\textit{Burkholderia}\xspace}
\newcommand{\vib}{\textit{Vibrio}\xspace}
\newcommand{\sul}{\textit{Sulfolobus}\xspace}
\newcommand{\ent}{\textit{Enterobacteria}\xspace}
\newcommand{\strep}{\textit{Streptomyces}\xspace}
\newcommand{\escoli}{\textit{Escherichia coli}\xspace}
\newcommand{\salm}{\textit{Salmonella enterica}\xspace}
\newcommand{\ecol}{\textit{E.\,coli}\xspace}
\newcommand{\bas}{\textit{Bacillus subtilis}\xspace}
\newcommand{\lis}{\textit{Listeria monocytogenes}\xspace}
\newcommand{\bass}{\textit{B.\,subtilis}\xspace}
\newcommand{\p}{progressiveMauve\xspace}
\newcommand{\salb}{\textit{S.\,alblus}\xspace}
\newcommand{\salbus}{\textit{S.\,albus}\xspace}
\newcommand{\shyg}{\textit{S.\,hygroscopicus}\xspace}
\newcommand{\sliv}{\textit{S.\,lividans}\xspace}
\newcommand{\sros}{\textit{S.\,roseosporus}\xspace}
\newcommand{\ssir}{\textit{S.\,sirex}\xspace}
\newcommand{\sven}{\textit{S.\,venezuelae}\xspace}
\newcommand{\scoe}{\textit{S.\,coelicolor}\xspace}
\newcommand{\tub}{\textit{Mycobacterium tuberculosis}\xspace}
\newcommand{\hal}{\textit{Haloquadratum walsbyi}\xspace}
\providecommand{\e}[1]{\ensuremath{\times 10^{#1}}}
\newcommand{\beginsupplement}{%
	\setcounter{table}{0}
	\renewcommand{\thetable}{S\arabic{table}}%
	\setcounter{figure}{0}
	\renewcommand{\thefigure}{S\arabic{figure}}%
}
\renewcommand{\thesection}{}
\renewcommand{\thesubsection}{}
\renewcommand{\thesubsubsection}{}
\usepackage{setspace}

\begin{document}
	\thispagestyle{empty}
	\maketitle
	\thispagestyle{empty}	
	\newpage
%	\thispagestyle{empty}
%	\tableofcontents
%	\newpage
%	\onehalfspacing
	\doublespacing
	
	\section{Abstract}

	\section{Materials and Methods}
	
	\section{Results}
	
	\section{Discussion}

\section{Acknowledgements}	
%We thank Caitlin Simopoulos for comments on the manuscript.

	\newpage
	
	\bibliographystyle{chicago}
	\bibliography{C:/Users/Daniella/Documents/Sinorhizobium2015/Bib/Bibliography}
	
	\newpage
	
	\section{Figures}
\newpage
	\section{Supplementary Figures}
	\beginsupplement
	
	\section*{Logistic Regression}
\begin{table}[H]
	\begin{center}
		\scalebox{0.68}{
			\begin{tabular}{llcc}
				\toprule
				COG Abbreviation & COG Category & \ecol Chromosome & \bass Chromosome \\
				\midrule
				C & Energy Production and Conversion &  2.40\e{-7} & 4.10\e{-7}\\
				E & Amino Acid Transport and Metabolism & 4.53\e{-7} & NA \\
				F & Nucleotide Transport and Metabolism & NA & \cellcolor{black!16}-7.49\e{-7} \\
				G & Carbohydrate Transport and Metabolism & NA & NA \\
				H & Coenzyme Transport and Metabolism & NA & \cellcolor{black!16}-4.07\e{-7}\\
				I & Lipid Transport and Metabolism & 4.51\e{-7} & 3.74\e{-7} \\
				J & Translation, Ribosomal Structure and Biogenesis & NA  & \cellcolor{black!16}-7.13\e{-7}\\
				K & Transcription & 2.22\e{-7} & 7.62\e{-7} \\
				L & Repliction, Recombination and Repair & 2.95\e{-7} & NA\\
				M & Cell Wall/Membrane/Envelope Biogenesis & NA & 5.18\e{-7}\\
				N & Cell Motility & \cellcolor{black!16}-7.74\e{-7}  & 1.01\e{-6}\\
				O & Posttranslational Modification, Protein Tunover, Chaperones & 3.37\e{-7} & 3.51\e{-7} \\
				P & Inorganic Ion Transport and Metabolism & NA & NA\\
				Q & Secondary Metabolites Biosynthesis, Transport and Catabolism & NA & NA\\
				T & Signl Transduction Mechanisms & NA & NA\\
				U & Intracellular Trafficking, Secretion, and Vesicular Transport & NA & NA \\
				V & Defense Mechanisms & 3.75\e{-7} & 7.15\e{-7}\\
				W & Extracellular Structures & \cellcolor{black!16}-3.23\e{-6} & NA\\
				X & Mobilome: Prophages, Transposons & \cellcolor{black!16}-1.09\e{-6} & \cellcolor{black!16}-1.81\e{-6}\\
				R & General Function Prediction Only & 2.61\e{-7} & 3.20\e{-7}\\
				S & Function Unknown & \cellcolor{black!16}-1.43\e{-6} & \cellcolor{black!16}1.19\e{-6}\\
				\bottomrule
			\end{tabular}}
		\end{center}
	\end{table}


%	\section*{Logistic Regression Stats}
%\begin{table}[h]
%	\begin{center}
%		\scalebox{0.68}{
%			\begin{tabular}{lllll}
%				\toprule
%				\multicolumn{5}{c}{\ecoli} \\
%				\midrule
%				COG Abbreviation & COG Category & Coefficient Estimate & Standard Error & P-value \\
%				\midrule
%				C & Energy Production and Conversion & 2.396\e{-7} & 7.202\e{-8} & 8.77\e{-4} \\
%				E & Amino Acid Transport and Metabolism & 4.53\e{-7} & 6.464\e{-8} & 2.42\e{-12} \\
%				I & Lipid Transport and Metabolism & 4.506\e{-7} & 1.065\e{-7} & 2.32\e{-5} \\
%				K & Transcription & 2.221\e{-7} & 7.031\e{-8} & 1.58\e{-3}\\
%				L & Repliction, Recombination and Repair & 2.95\e{-7} & 9.903\e{-8} & 2.89\e{-3} \\
%				\cellcolor{teal!55}N & \cellcolor{teal!55}Cell Motility & \cellcolor{teal!55}-7.743\e{-7} & \cellcolor{teal!55}1.453\e{-7} & \cellcolor{teal!55}9.8\e{-8}\\
%				O & Posttranslational Modification, Protein Tunover, Chaperones & 3.371\e{-7} & 9.418\e{-8} & 3.45\e{-4}\\
%				R & General Function Prediction Only & 2.606\e{-7} & 7.056\e{-8} & 2.21\e{-4}\\
%				\cellcolor{teal!55}S & \cellcolor{teal!55}Function Unknown & \cellcolor{teal!55}-1.427\e{-6} & \cellcolor{teal!55}1.323\e{-7} & \cellcolor{teal!55}$<$2\e{-16}\\
%				V & Defense Mechanisms & 3.754\e{-7} & 1.203\e{-7} & 1.8\e{-3}\\
%				\cellcolor{teal!55}W & \cellcolor{teal!55}Extracellular Structures & \cellcolor{teal!55}-3.234\e{-6} & \cellcolor{teal!55}5.718\e{-7} & \cellcolor{teal!55}1.55\e{-8}\\
%				\cellcolor{teal!55}X & \cellcolor{teal!55}Mobilome: Prophages, Transposons & \cellcolor{teal!55}-1.085\e{-6} & \cellcolor{teal!55}1.962\e{-7} & \cellcolor{teal!55}3.23\e{-8}\\
%				\bottomrule
%			\end{tabular}}
%		\end{center}
%	\end{table}
%	
%	\begin{table}[h]
%		\begin{center}
%			\scalebox{0.68}{
%				\begin{tabular}{lllll}
%					\toprule
%					\multicolumn{5}{c}{\bass} \\
%					\midrule
%					COG Abbreviation & COG Category & Coefficient Estimate & Standard Error & P-value \\
%					\midrule
%					C & Energy Production and Conversion & 4.095\e{-7} & 1.284\e{-7} & 1.42\e{-3} \\
%					\cellcolor{teal!55}F & \cellcolor{teal!55}Nucleotide Transport and Metabolism & \cellcolor{teal!55}-7.49\e{-7} & \cellcolor{teal!55}1.771\e{-7} & \cellcolor{teal!55}2.35\e{-5} \\
%					\cellcolor{teal!55}H & \cellcolor{teal!55}Coenzyme Transport and Metabolism & \cellcolor{teal!55}-4.068\e{-7} & \cellcolor{teal!55}1.223\e{-7} & \cellcolor{teal!55}8.77\e{-4} \\
%					I & Lipid Transport and Metabolism & 3.744\e{-7} & 1.358\e{-7} & 5.84\e{-3} \\
%					\cellcolor{teal!55}J & \cellcolor{teal!55}Translation, Ribosomal Structure and Biogenesis & \cellcolor{teal!55}-7.131\e{-7} & \cellcolor{teal!55}1.171\e{-7} & \cellcolor{teal!55}1.12\e{-9} \\
%					K & Transcription & 7.615\e{-7} & 1.001\e{-7} & 2.83\e{-14}\\
%					M & Cell Wall/Membrane/Envelope Biogenesis & 5.182\e{-7} & 1.091\e{-7} & 2.02\e{-6} \\
%					N & Cell Motility & 1.009\e{-6} & 2.191\e{-7} & 4.11\e{-6}\\
%					O & Posttranslational Modification, Protein Tunover, Chaperones & 3.508\e{-7} & 1.451\e{-8} & 1.56\e{-2}\\
%					R & General Function Prediction Only & 3.197\e{-7} & 8.967\e{-8} & 3.63\e{-4}\\
%					\cellcolor{teal!55}S & \cellcolor{teal!55}Function Unknown & \cellcolor{teal!55}-1.185\e{-6} & \cellcolor{teal!55}1.286\e{-7} & \cellcolor{teal!55}$<$2\e{-16}\\
%					V & Defense Mechanisms & 7.148\e{-7} & 1.711\e{-7} & 2.94\e{-5}\\
%					%				\cellcolor{teal!55}W & \cellcolor{teal!55}Extracellular Structures & \cellcolor{teal!55}-3.234\e{-6} & \cellcolor{teal!55}5.718\e{-7} & \cellcolor{teal!55}1.55\e{-8}\\
%					\cellcolor{teal!55}X & \cellcolor{teal!55}Mobilome: Prophages, Transposons & \cellcolor{teal!55}-1.814\e{-6} & \cellcolor{teal!55}4.027\e{-7} & \cellcolor{teal!55}6.65\e{-6}\\
%					\bottomrule
%				\end{tabular}}
%			\end{center}
%		\end{table}
%		
%		
%		\begin{table}[h]
%			\begin{center}
%				\scalebox{0.68}{
%					\begin{tabular}{lllll}
%						\toprule
%						\multicolumn{5}{c}{\strep} \\
%						\midrule
%						COG Abbreviation & COG Category & Coefficient Estimate & Standard Error & P-value \\
%						\midrule
%						\cellcolor{teal!55}C & \cellcolor{teal!55}Energy Production and Conversion & \cellcolor{teal!55}-2.939\e{-7} & \cellcolor{teal!55}2.536\e{-8} & \cellcolor{teal!55}$<$2\e{-16} \\
%						\cellcolor{teal!55}E & \cellcolor{teal!55}Amino Acid Transport and Metabolism & \cellcolor{teal!55}-1.967\e{-7} & \cellcolor{teal!55}1.849\e{-8} & \cellcolor{teal!55}$<$2\e{-12} \\
%						\cellcolor{teal!55}F & \cellcolor{teal!55}Nucleotide Transport and Metabolism & \cellcolor{teal!55}-1.864\e{-7} & \cellcolor{teal!55}4.142\e{-8} & \cellcolor{teal!55}6.78\e{-6} \\
%						\cellcolor{teal!55}H & \cellcolor{teal!55}Coenzyme Transport and Metabolism & \cellcolor{teal!55}-1.112\e{-7} & \cellcolor{teal!55}2.172\e{-8} & \cellcolor{teal!55}3.06\e{-7} \\
%						\cellcolor{teal!55}I & \cellcolor{teal!55}Lipid Transport and Metabolism & \cellcolor{teal!55}-2.011\e{-7} & \cellcolor{teal!55}1.952\e{-8} & \cellcolor{teal!55}$<$2\e{-3} \\
%						\cellcolor{teal!55}J & \cellcolor{teal!55}Translation, Ribosomal Structure and Biogenesis & \cellcolor{teal!55}-1.355\e{-7} & \cellcolor{teal!55}2.59\e{-8} & \cellcolor{teal!55}1.68\e{-7} \\
%						%				K & Transcription & 7.615\e{-7} & 1.001\e{-7} & 2.83\e{-14}\\
%						\cellcolor{teal!55}L & \cellcolor{teal!55}Repliction, Recombination and Repair & \cellcolor{teal!55}-1.174\e{-7} & \cellcolor{teal!55}3.205\e{-8} & \cellcolor{teal!55}2.5\e{-4} \\
%						\cellcolor{teal!55}M & \cellcolor{teal!55}Cell Wall/Membrane/Envelope Biogenesis & \cellcolor{teal!55}-8.046\e{-8} & \cellcolor{teal!55}2.326\e{-8} & \cellcolor{teal!55}5.43\e{-4} \\
%						\cellcolor{teal!55}N & \cellcolor{teal!55}Cell Motility & \cellcolor{teal!55}-2.04\e{-7} & \cellcolor{teal!55}9.227\e{-8} & \cellcolor{teal!55}2.71\e{-2}\\
%						\cellcolor{teal!55}O & \cellcolor{teal!55}Posttranslational Modification, Protein Tunover, Chaperones & \cellcolor{teal!55}-7.751\e{-8} & \cellcolor{teal!55}2.777\e{-8} & \cellcolor{teal!55}5.25\e{-3}\\
%						\cellcolor{teal!55}P & \cellcolor{teal!55}Inorganic Ion Transport and Metabolism & \cellcolor{teal!55}-1.682\e{-7} & \cellcolor{teal!55}2.346\e{-8} & \cellcolor{teal!55}7.52\e{-13} \\
%						\cellcolor{teal!55}R & \cellcolor{teal!55}General Function Prediction Only & \cellcolor{teal!55}-6.914\e{-8} & \cellcolor{teal!55}1.285\e{-8} & \cellcolor{teal!55}7.38\e{-8}\\
%						S & Function Unknown & 4.615\e{-7} & 2.671\e{-8} & $<$2\e{-16}\\
%						T & Signl Transduction Mechanisms & 1.517\e{-7} & 1.531\e{-8} & $<$2\e{-16} \\
%						\cellcolor{teal!55}V & \cellcolor{teal!55}Defense Mechanisms & \cellcolor{teal!55}-1.205\e{-7} & \cellcolor{teal!55}2.739\e{-8} & \cellcolor{teal!55}1.09\e{-5}\\
%						W & Extracellular Structures & 9.063\e{-7} & 2.336\e{-7} & 1.05\e{-4}\\
%						X & Mobilome: Prophages, Transposons & 4.323\e{-7} & 3.599\e{-8} & $<$2\e{-16}\\
%						\bottomrule
%					\end{tabular}}
%				\end{center}
%			\end{table}
%			
%			\begin{table}[h]
%				\begin{center}
%					\scalebox{0.68}{
%						\begin{tabular}{lllll}
%							\toprule
%							\multicolumn{5}{c}{\smel Chromosome} \\
%							\midrule
%							COG Abbreviation & COG Category & Coefficient Estimate & Standard Error & P-value \\
%							\midrule
%							%					\cellcolor{teal!55}C & \cellcolor{teal!55}Energy Production and Conversion & \cellcolor{teal!55}-2.939\e{-7} & \cellcolor{teal!55}2.536\e{-8} & \cellcolor{teal!55}$<$2\e{-16} \\
%							E & Amino Acid Transport and Metabolism & 2.662\e{-7} & 8.996\e{-8} & 3.08\e{-3} \\
%							\cellcolor{teal!55}F & \cellcolor{teal!55}Nucleotide Transport and Metabolism & \cellcolor{teal!55}-6.682\e{-7} & \cellcolor{teal!55}2.02\e{-7} & \cellcolor{teal!55}9.43\e{-4} \\
%							\cellcolor{teal!55}G & \cellcolor{teal!55}Carbohydrate Transport and Metabolism & \cellcolor{teal!55}-2.533\e{-7} & \cellcolor{teal!55}1.067\e{-7} & \cellcolor{teal!55}1.76\e{-2} \\
%							\cellcolor{teal!55}H & \cellcolor{teal!55}Coenzyme Transport and Metabolism & \cellcolor{teal!55}-1.095\e{-6} & \cellcolor{teal!55}1.574\e{-7} & \cellcolor{teal!55}3.5\e{-12} \\
%							%					\cellcolor{teal!55}I & \cellcolor{teal!55}Lipid Transport and Metabolism & \cellcolor{teal!55}-2.011\e{-7} & \cellcolor{teal!55}1.952\e{-8} & \cellcolor{teal!55}$<$2\e{-3} \\
%							\cellcolor{teal!55}J & \cellcolor{teal!55}Translation, Ribosomal Structure and Biogenesis & \cellcolor{teal!55}-1.233\e{-6} & \cellcolor{teal!55}1.544\e{-7} & \cellcolor{teal!55}1.42\e{-15} \\
%							%				K & Transcription & 7.615\e{-7} & 1.001\e{-7} & 2.83\e{-14}\\
%							%					\cellcolor{teal!55}L & \cellcolor{teal!55}Repliction, Recombination and Repair & \cellcolor{teal!55}-1.174\e{-7} & \cellcolor{teal!55}3.205\e{-8} & \cellcolor{teal!55}2.5\e{-4} \\
%							M & Cell Wall/Membrane/Envelope Biogenesis & 4.594\e{-7} & 1.38\e{-7} & 8.73\e{-4} \\
%							%					\cellcolor{teal!55}N & \cellcolor{teal!55}Cell Motility & \cellcolor{teal!55}-2.04\e{-7} & \cellcolor{teal!55}9.227\e{-8} & \cellcolor{teal!55}2.71\e{-2}\\
%							O & Posttranslational Modification, Protein Tunover, Chaperones & 3.474\e{-7} & 1.623\e{-7} & 3.23\e{-2}\\
%							P & Inorganic Ion Transport and Metabolism & 5.361\e{-7} & 1.367\e{-7} & 8.77\e{-5} \\
%							R & General Function Prediction Only & 9.493\e{-7} & 1.07\e{-7} & $<$2\e{-16}\\
%							\cellcolor{teal!55}S & \cellcolor{teal!55}Function Unknown & \cellcolor{teal!55}-7.436\e{-7} & \cellcolor{teal!55}3.329\e{-7} & \cellcolor{teal!55}2.55\e{-2}\\
%							T & Signl Transduction Mechanisms & 1.851\e{-6} & 1.948\e{-7} & $<$2\e{-16} \\
%							U & Intracellular Trafficking, Secretion, and Vesicular Transport & 8.624\e{-7} & 4.29\e{-7} & 4.44\e{-2} \\
%							V & Defense Mechanisms & 4.235\e{-7} & 2.036\e{-7} & 3.75\e{-2}\\
%							%					W & Extracellular Structures & 9.063\e{-7} & 2.336\e{-7} & 1.05\e{-4}\\
%							X & Mobilome: Prophages, Transposons & 1.673\e{-6} & 6.938\e{-7} & 1.59\e{-2}\\
%							\bottomrule
%						\end{tabular}}
%					\end{center}
%				\end{table}
%				
%				\begin{table}[h]
%					\begin{center}
%						\scalebox{0.68}{
%							\begin{tabular}{lllll}
%								\toprule
%								\multicolumn{5}{c}{\smel pSymA} \\
%								\midrule
%								COG Abbreviation & COG Category & Coefficient Estimate & Standard Error & P-value \\
%								\midrule
%								%					\cellcolor{teal!55}C & \cellcolor{teal!55}Energy Production and Conversion & \cellcolor{teal!55}-2.939\e{-7} & \cellcolor{teal!55}2.536\e{-8} & \cellcolor{teal!55}$<$2\e{-16} \\
%								\cellcolor{teal!55}E & \cellcolor{teal!55}Amino Acid Transport and Metabolism & \cellcolor{teal!55}-2.077\e{-6} & \cellcolor{teal!55}3.619\e{-7} & \cellcolor{teal!55}9.45\e{-9} \\
%								\cellcolor{teal!55}F & \cellcolor{teal!55}Nucleotide Transport and Metabolism & \cellcolor{teal!55}-1.147\e{-6} & \cellcolor{teal!55}5.704\e{-7} & \cellcolor{teal!55}4.43\e{-2} \\
%								G & Carbohydrate Transport and Metabolism & 9.779\e{-7} & 4.287\e{-7} & 2.25\e{-2} \\
%								\cellcolor{teal!55}H & \cellcolor{teal!55}Coenzyme Transport and Metabolism & \cellcolor{teal!55}-9.828\e{-7} & \cellcolor{teal!55}4.699\e{-7} & \cellcolor{teal!55}3.65\e{-2} \\
%								I & Lipid Transport and Metabolism & 2.006\e{-6} & 6.299\e{-7} & 1.45\e{-3} \\
%								\cellcolor{teal!55}J & \cellcolor{teal!55}Translation, Ribosomal Structure and Biogenesis & \cellcolor{teal!55}-1.506\e{-6} & \cellcolor{teal!55}4.205\e{-7} & \cellcolor{teal!55}3.42\e{-4} \\
%								%				K & Transcription & 7.615\e{-7} & 1.001\e{-7} & 2.83\e{-14}\\
%								L & Repliction, Recombination and Repair & 1.416\e{-6} & 6.18\e{-7} & 2.19\e{-2} \\
%								M & Cell Wall/Membrane/Envelope Biogenesis & 1.633\e{-6} & 5.42\e{-7} & 2.58\e{-3} \\
%								%					\cellcolor{teal!55}N & \cellcolor{teal!55}Cell Motility & \cellcolor{teal!55}-2.04\e{-7} & \cellcolor{teal!55}9.227\e{-8} & \cellcolor{teal!55}2.71\e{-2}\\
%								%							O & Posttranslational Modification, Protein Tunover, Chaperones & 3.474\e{-7} & 1.623\e{-7} & 3.23\e{-2}\\
%								%							P & Inorganic Ion Transport and Metabolism & 5.361\e{-7} & 1.367\e{-7} & 8.77\e{-5} \\
%								Q & Secondary Metabolites Biosynthesis, Transport and Catabolism & 4.283\e{-6} & 6.738\e{-7} & 2.07\e{-10} \\
%								%							R & General Function Prediction Only & 9.493\e{-7} & 1.07\e{-7} & $<$2\e{-16}\\
%								S & Function Unknown & 2.53\e{-5} & 9.966\e{-6} & 1.111\e{-2}\\
%								%							T & Signl Transduction Mechanisms & 1.851\e{-6} & 1.948\e{-7} & $<$2\e{-16} \\
%								%							U & Intracellular Trafficking, Secretion, and Vesicular Transport & 8.624\e{-7} & 4.29\e{-7} & 4.44\e{-2} \\
%								%							V & Defense Mechanisms & 4.235\e{-7} & 2.036\e{-7} & 3.75\e{-2}\\
%								%					W & Extracellular Structures & 9.063\e{-7} & 2.336\e{-7} & 1.05\e{-4}\\
%								%							X & Mobilome: Prophages, Transposons & 1.673\e{-6} & 6.938\e{-7} & 1.59\e{-2}\\
%								\bottomrule
%							\end{tabular}}
%						\end{center}
%					\end{table}
%					
%					\begin{table}[h]
%						\begin{center}
%							\scalebox{0.68}{
%								\begin{tabular}{lllll}
%									\toprule
%									\multicolumn{5}{c}{\smel pSymB} \\
%									\midrule
%									COG Abbreviation & COG Category & Coefficient Estimate & Standard Error & P-value \\
%									\midrule
%									%					\cellcolor{teal!55}C & \cellcolor{teal!55}Energy Production and Conversion & \cellcolor{teal!55}-2.939\e{-7} & \cellcolor{teal!55}2.536\e{-8} & \cellcolor{teal!55}$<$2\e{-16} \\
%									\cellcolor{teal!55}E & \cellcolor{teal!55}Amino Acid Transport and Metabolism & \cellcolor{teal!55}-9.452\e{-7} & \cellcolor{teal!55}2.901\e{-7} & \cellcolor{teal!55}1.12\e{-3} \\
%									%							\cellcolor{teal!55}F & \cellcolor{teal!55}Nucleotide Transport and Metabolism & \cellcolor{teal!55}-1.147\e{-6} & \cellcolor{teal!55}5.704\e{-7} & \cellcolor{teal!55}4.43\e{-2} \\
%									G & Carbohydrate Transport and Metabolism & 2.048\e{-6} & 4.07\e{-7} & 4.87\e{-7} \\
%									\cellcolor{teal!55}H & \cellcolor{teal!55}Coenzyme Transport and Metabolism & \cellcolor{teal!55}-1.452\e{-6} & \cellcolor{teal!55}4.089\e{-7} & \cellcolor{teal!55}3.84\e{-4} \\
%									I & Lipid Transport and Metabolism & 1.839\e{-6} & 5.832\e{-7} & 1.61\e{-3} \\
%									\cellcolor{teal!55}J & \cellcolor{teal!55}Translation, Ribosomal Structure and Biogenesis & \cellcolor{teal!55}-1.151\e{-6} & \cellcolor{teal!55}3.589\e{-7} & \cellcolor{teal!55}1.34\e{-3} \\
%									\cellcolor{teal!55}K & \cellcolor{teal!55}Transcription & \cellcolor{teal!55}-4.166\e{-6} & \cellcolor{teal!55}5.168\e{-7} & \cellcolor{teal!55}7.59\e{-16}\\
%									%							L & Repliction, Recombination and Repair & 1.416\e{-6} & 6.18\e{-7} & 2.19\e{-2} \\
%									M & Cell Wall/Membrane/Envelope Biogenesis & 5.408\e{-6} & 7.131\e{-7} & 3.37\e{-14} \\
%									%					\cellcolor{teal!55}N & \cellcolor{teal!55}Cell Motility & \cellcolor{teal!55}-2.04\e{-7} & \cellcolor{teal!55}9.227\e{-8} & \cellcolor{teal!55}2.71\e{-2}\\
%									O & Posttranslational Modification, Protein Tunover, Chaperones & 1.078\e{-6} & 5.23\e{-7} & 3.93\e{-2}\\
%									\cellcolor{teal!55}P & \cellcolor{teal!55}Inorganic Ion Transport and Metabolism & \cellcolor{teal!55}-2.045\e{-6} & \cellcolor{teal!55}4.404\e{-7} & \cellcolor{teal!55}3.41\e{-6} \\
%									Q & Secondary Metabolites Biosynthesis, Transport and Catabolism & 3.806\e{-6} & 6.679\e{-7} & 1.21\e{-8} \\
%									%							R & General Function Prediction Only & 9.493\e{-7} & 1.07\e{-7} & $<$2\e{-16}\\
%									S & Function Unknown & 4.141\e{-5} & 1.595\e{-5} & 9.44\e{-3}\\
%									%							T & Signl Transduction Mechanisms & 1.851\e{-6} & 1.948\e{-7} & $<$2\e{-16} \\
%									%							U & Intracellular Trafficking, Secretion, and Vesicular Transport & 8.624\e{-7} & 4.29\e{-7} & 4.44\e{-2} \\
%									%							V & Defense Mechanisms & 4.235\e{-7} & 2.036\e{-7} & 3.75\e{-2}\\
%									%					W & Extracellular Structures & 9.063\e{-7} & 2.336\e{-7} & 1.05\e{-4}\\
%									\cellcolor{teal!55}X & \cellcolor{teal!55}Mobilome: Prophages, Transposons & \cellcolor{teal!55}-3.347\e{-6} & \cellcolor{teal!55}1.514\e{-6} & \cellcolor{teal!55}2.7\e{-2}\\
%									\bottomrule
%								\end{tabular}}
%							\end{center}
%						\end{table}
%						%\end{landscape}
%
%	
\end{document}
